test/integration/src/test/java/io/pravega/test/integration/endtoendtest/EndToEndReaderGroupTest.java

```diff
@@ -204,6 +165,58 @@ public void testMultiScopeReaderGroup() throws Exception {
         assertTrue(managedStreams.contains(Stream.of(scopeB, streamName).getScopedName()));
     }
 
+    @Test(timeout = 30000)
+    public void getCurrentStreamCutTest() throws Exception {
+        final Stream stream = Stream.of(SCOPE, STREAM);
+        final String group = "group";
+
+        createScope(SCOPE);
+        createStream(SCOPE, STREAM, ScalingPolicy.fixed(1));
```

Comment:
Perhaps also test with more segments per stream and even in the presence of scaling?

Reply:
Done. Added a new test case.